.github/workflows/automate-changelog-PR.yml

```diff
@@ -0,0 +1,48 @@
+name: "Update changelog and open a pull request"
+
+on:
+  workflow_dispatch:
+    inputs:
+      fname_changelog:
+        description: 'Changelog filename'
+        required: true
+        default: 'CHANGES.md'
+      milestone_title:
+        description: 'Milestone title'
+        required: true
+
+jobs:
+  create-changelog-PR:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Set up Python (to install 'changelog')
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+    - name: Checkout 'changelog'
+      uses: actions/checkout@v2
+      with:
+        repository: neuropoly/changelog
+        path: changelog
+    - name: Install 'changelog'
+      run: |
+        pip install ./changelog
+    - name: Checkout '${{ github.event.repository.name }}'
+      uses: actions/checkout@v2
+      with:
+        path: ${{ github.event.repository.name }}
+    - name: Update '${{ github.event.inputs.fname_changelog }}' for '${{ github.event.repository.name }}'
+      run: |
+        cd ${{ github.event.repository.name }}
+        changelog neuropoly/spinalcordtoolbox --update --name "${{ github.event.inputs.fname_changelog }}" --milestone ${{ github.event.inputs.milestone_title }}
+        rm "${{ github.event.inputs.fname_changelog }}.bak"
+    - name: Create pull request for updated changelog
+      uses: peter-evans/create-pull-request@v3
+      with:
+        path: "${{ github.event.repository.name }}"
+        branch: "bot/v${{ github.event.inputs.milestone_title }}"
+        commit-message  : "Update ${{ github.event.inputs.fname_changelog }} for ${{ github.event.inputs.milestone_title }} release"
+        title: "Update ${{ github.event.inputs.fname_changelog }} for ${{ github.event.inputs.milestone_title }} release"
+        milestone: "${{ github.event.inputs.milestone_title }}"
+        body: "Automated step carried out as a part of [Creating a new release](https://github.com/neuropoly/spinalcordtoolbox/wiki/Misc%3A-Creating-a-new-release) checklist."
```

Comment:
One of the examples you made with this, https://github.com/joshuacwnewton/test-github-actions/pull/16, looks like

![2021-04-07-115012_781x460_scrot](https://user-images.githubusercontent.com/987487/113895911-6f69e980-9797-11eb-9edd-92d68284217b.png)

I don't think you need to repeat "Automated", since Github already tags it with "bot", but my eye skipped over the title when looking at this. How about this wording:


```suggestion
        body: |
          Update ${{ github.event.inputs.fname_changelog }} for ${{ github.event.inputs.milestone_title }} release.
          
          Carried out as a part of the [release](https://github.com/neuropoly/spinalcordtoolbox/wiki/Misc%3A-Creating-a-new-release) checklist."
```